migtests/scripts/live-migration-fallf-run-test.sh

```diff
@@ -51,20 +52,29 @@ main() {
 
 	pushd ${TEST_DIR}
 
-	step "Setup Fall Forward environment"
-	create_ff_schema ${SOURCE_REPLICA_DB_NAME}
-	run_sqlplus_as_sys ${SOURCE_REPLICA_DB_NAME} ${SCRIPTS}/oracle/create_metadata_tables.sql
+	if [ "${SOURCE_DB_TYPE}" = "oracle" ]
+	then
+		step "Setup Fall Forward environment for Oracle"
+		create_ff_schema ${SOURCE_REPLICA_DB_NAME}
+		run_sqlplus_as_sys ${SOURCE_REPLICA_DB_NAME} ${SCRIPTS}/oracle/create_metadata_tables.sql
+	fi
 
 	step "Initialise source and fall forward database."
 	./init-db
 
-	step "Grant source database user permissions"
+	step "Grant source database user permissions for live migration"
 	if [ "${SOURCE_DB_TYPE}" = "oracle" ]
 	then
 		grant_permissions_for_live_migration_oracle ${ORACLE_CDB_NAME} ${SOURCE_DB_NAME}
 		run_sqlplus_as_sys ${SOURCE_REPLICA_DB_NAME} ${SCRIPTS}/oracle/fall_forward_prep.sql
-	else
-		grant_permissions ${SOURCE_DB_NAME} ${SOURCE_DB_TYPE} ${SOURCE_DB_SCHEMA}
+	elif [ "${SOURCE_DB_TYPE}" = "postgresql" ]
+	then
+		set_replica_identity
```

Comment:
How come you don't have to do any of the publication stuff?  Are you using superuser?

Reply:
Yes using superuser for now. Will be adding proper permissions later in a separate PR.